libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.ts

```diff
@@ -26,6 +26,7 @@ import {
 } from '@perun-web-apps/perun/services';
 import { merge } from 'rxjs';
 import { tap } from 'rxjs/operators';
+import { TableWrapperComponent } from '../table-wrapper/table-wrapper.component';
```

Comment:
```suggestion
import { TableWrapperComponent } from '@perun-web-apps/perun/components';
```
Idea told me that you can shorten the import
do it on every table